git_machete/tests/functional/test_machete.py

```diff
@@ -92,6 +93,12 @@ def launch_command(*args: str) -> str:
                 git.flush_caches()
             return out.getvalue()
 
+    @staticmethod
+    def edit_definition_file(new_body: List[str]) -> None:
```

Comment:
I'd say:
```suggestion
    def rewrite_definition_file(new_lines: List[str]) -> None:
```

Since `edit` suggests sth like `git machete edit`, an editor being opened